test/resources/theme-ugly.css

```diff
@@ -1,5 +1,6 @@
 :root {
     --fd-color-action-1: magenta;
+    --fd-color-action: orangered;
```

Comment:
Do we really have CSS variables names `--fd-color-action` and then `--fd-color-action-1`?

Reply:
It depends on the component. For button they use the variables you defined previously (the ones with a number appended), but for more simple components such as the label, it is `--fd-color-action` (without a number appended) that is used.

Reply:
We need to tackle the colors, but it can be done separately.
We can just import the sass(and css variable) file from the theme designer and use only those variables.